webpack.config.js

```diff
@@ -0,0 +1,106 @@
+const path = require('path');
+const argv = require('yargs').argv;
+const UglifyJSPlugin = require('uglifyjs-webpack-plugin');
+const webpack = require('webpack');
+
+
+let FILE_NAME = 'cross-domain-utils';
+let MODULE_NAME = 'crossDomainUtils';
+
+export let defaultWebpackConfig = {
+    stats: process.env.NODE_ENV === 'debug' ? "verbose" : "normal",
+    bail: true,
+    entry: './src/index.js',
+    module: {
+        rules: [ {
+            test: /\.js$/,
+            exclude: /(node_modules)/,
+            loader: 'babel-loader',
+            options: {
+                presets: ["es2015"]
+            }
+        } ]
+    },
+    resolve: {
+        modules: [
+            __dirname,
+            'node_modules'
+        ]
+    },
+    plugins: [
+        new webpack.DefinePlugin({
+            'process.env.NODE_ENV': JSON.stringify(process.env.NODE_ENV)
+        })
+    ]
+};
+
+export let testConfig = Object.assign({}, defaultWebpackConfig, {
+    devtool: 'inline-source-map',
+    node: { // WORKAROUND: https://github.com/webpack-contrib/css-loader/issues/447
+        fs: 'empty',
+        child_process: 'empty'
+    },
+    module: {
+        rules: [
+            {
+                test: /\.js$/,
+                exclude: /(node_modules)/,
+                loader: 'babel-loader',
+                options: {
+                    presets: ["es2015"]
+                }
+            },
+            
+            {
+                test: /\.js$/,
+                // enforce: 'post',
+                include: path.resolve('src/'),
+                exclude: /node_modules|test/,
+                use: {
+                    loader: 'istanbul-instrumenter-loader',
+                    options: {
+                        esModules: true
+                    }
+                }
+            }
+        ]
+    }
+});
+
+export let webConfig = Object.assign({}, defaultWebpackConfig, {
+    target: 'web',
+    devtool: 'source-map',
+    output: {
+        filename: FILE_NAME + '.js',
+        pathinfo: false,
+        library: MODULE_NAME
+    },
+    plugins: [
+        new webpack.optimize.ModuleConcatenationPlugin(),
```

Comment:
Asked a few times but didn't see any response. Did you try loading the `dist/cross-domain-utils.js` file directly in the browser? As I mentioned, this plugin broke checkout.js so want to make sure we avoid the same problem here.